libraries/AP_OSD/AP_OSD.cpp

```diff
@@ -85,7 +85,11 @@ const AP_Param::GroupInfo AP_OSD::var_info[] = {
     // @Param: _OPTIONS
     // @DisplayName: OSD Options
     // @Description: This sets options that change the display
+#if OSD_CRSF_PANELS_ENABLED
```

Comment:
I think we now prefer an AP prefix so AP_OSD_CRSF_PANELS_ENABLED

Reply:
I think you should just unilaterally add the docs, they will at least then show up in mission planner. We do this in other places as well.

Reply:
Ok, I have done these changes in the latest commits.